odxtools/compumethods/compuscale.py

```diff
@@ -64,4 +74,35 @@ def from_et(et_element: ElementTree.Element, doc_frags: List[OdxDocFragment], *,
             upper_limit=upper_limit,
             compu_inverse_value=compu_inverse_value,
             compu_const=compu_const,
-            compu_rational_coeffs=compu_rational_coeffs)
+            compu_rational_coeffs=compu_rational_coeffs,
+            internal_type=internal_type,
+            physical_type=physical_type)
+
+    def applies(self, internal_value: AtomicOdxType) -> bool:
+
+        if self.lower_limit is None and self.upper_limit is None:
```

Comment:
limits are required?

Reply:
the XSD says that they are optional for COMPU-SCALE. (this is part of why handling of limits is so awkward...)

Reply:
this is the compu scale class... am I missing something?

Reply:
ah, okay. I suppose that I misread your point: this is supposed to be True?! (did I ever mention that limit handling is surprisingly difficult? anyway: fixed